<launch>

  <!-- 필요한 인자만 -->
  <arg name="scan_topic"  default="/scan"/>
  <arg name="params_file" default="$(find dogm_ros)/config/params.yaml"/>
  <arg name="base_frame"  default="base_link"/>
  <arg name="fixed_frame" default="odom"/>

  <!-- DOGM (로컬 그리드) 노드만 실행 -->
  <node pkg="dogm_ros" type="dogm_node" name="dogm_node" output="screen">
    <rosparam file="$(arg params_file)"/>
    <param name="lidar_topic" value="$(arg scan_topic)"/>
    <param name="base_frame"  value="$(arg base_frame)"/>
    <param name="fixed_frame" value="$(arg fixed_frame)"/>
  </node>


</launch>